libipl/p10/ipl0.C

```diff
@@ -30,6 +30,14 @@ extern "C" {
 #define FRU_TYPE_FC     0x53
 #define GUARD_ERROR_TYPE_RECONFIG  0xEB
 
+#define OSC_CTL_OFFSET      0x06
```

Comment:
Commit message should start with libipl: xxx

Provide details , like scope and test details.